### DISCUSSION TITLE
Add auther filter on "Module Manager" page

### DISCUSSION THREAD

**Discussion by dheerajwebkul** [PrestaShop/PrestaShop#33122] [Ideas]
### The problem

I'm always frustrated when developing a module, I always need to see our 'Webkul' module filtered on the "Module Manager" Page.

Please check screenshot: https://nimb.ws/8yPLa4

### The solution

Just provide a filter and keep page on filtered state always on load, add a remove filter button so that we can remove that filter.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by MatShir** [comment]
Would you be interested to contribute for this feature ? It seems a nice idea. We could add a sorting (sorting by name, module dev and by module categories)

**Comment by dheerajwebkul** [comment]
Sure, I am interested to contribute, I will create a PR request when done.

Thanks,

**Comment by boherm** [comment]
It could be a nice idea to improve module manager like this!
Thanks for your suggestion @dheerajwebkul 🙂

**Comment by kpodemski** [comment]
I like the idea! If you have any questions on contribution details, make sure to contact us here or on [Slack](https://www.prestashop-project.org/slack/).
